lib/export.js

```diff
@@ -423,7 +425,9 @@ function valueToHierarchyJSON(value) {
     result['label'] = identifierToString(value.identifier);
     result['identifier'] = identifierToHierarchyJSON(value.identifier);
   } else {
-    logger.error('Unknown type for value \'%s\'. ERROR_CODE:14002', value.constructor.name);
+    //14002 , 'Unknown type for value ${value1}' , 'Unknown' , 'errorNumber'
+    //logger.error('Unknown type for value \'%s\'. ERROR_CODE:14002', value.constructor.name);
+    logger.error({value1 : value.constructor.name }, '14002');

```

Comment:
Why do you name this `value1` and not just `value`?  Same for the change in line 491-- why is the key `constraint1` rather than just `constraint`?

Reply:
I see this is a pattern across all the modules, but I don't understand it...